### PR TITLE
start cheat-sheet as help during exercises

### PR DESCRIPTION
**Pull request by SepplL**
any important addition?

### CODE REVIEW

File: terminal-cheat-sheet/content/cheat-sheet.tex [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+\begin{frame}{Cheat-Sheet}
+  \begin{tblr}{
+      colspec = {rl},
+      column{1} = {font=\ttfamily, fg=vertexDarkRed}
+    }
+    ls & \enquote{list}: Zeigt den Inhalt eines Verzeichnisses \\
+    cd .. & \enquote{change directory} wechselt in den Oberordner \\
+    cd <dirname> & \enquote{change directory} wechselt in den angegebenen \\
+    mamba activate toolbox & laedt die installierten \texttt{python} Pakete \\
```
Comment (maxnoe) [inline terminal-cheat-sheet/content/cheat-sheet.tex:11]: lädt

**Comment by aknierim** [comment]
I would add `cd -` to get back to the last directory you came from

**Comment by maxnoe** [comment]
Maybe add a code example or screenshot how the Terminal looks like and how the ipython prompt looks like?

**Comment by SepplL** [comment]
> Maybe add a code example or screenshot how the Terminal looks like and how the ipython prompt looks like?

I'd rather have it as one compact slide on the wall during the exercises with not too much going on.
So no scrolling and no confusion with too much information and more as a 'look up' on the fly while the students try out things.

File: terminal-cheat-sheet/content/cheat-sheet.tex [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  \begin{tblr}{
+      colspec = {rl},
+      column{1} = {font=\ttfamily, fg=vertexDarkRed}
+    }
+    ls & \enquote{list}: Zeigt den Inhalt eines Verzeichnisses \\
+    cd .. & \enquote{change directory} wechselt in den Oberordner \\
+    cd - & \enquote{change directory} in das vorherige Verzeichnis \\
+    cd <dirname> & \enquote{change directory} wechselt in den angegebenen \\
+    mamba activate toolbox & lädt die installierten \texttt{python} Pakete \\
+    codium . & öffnet VSCodium im aktuellen Verzeichnis \\
```
Comment (jpherdi) [inline terminal-cheat-sheet/content/cheat-sheet.tex:13]:
```suggestion
    codium . & öffnet VSCodium im aktuellen Verzeichnis (Mac/Linux) \\
```

File: terminal-cheat-sheet/README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Cheat-Sheet
+
+In diesem Slide findet sich eine uebersicht der wichtigen Terminal Befehle wieder.
```
Comment (jpherdi) [inline terminal-cheat-sheet/README.md:3]:
```suggestion
Diese Folie zeigt eine Übersicht der wichtigsten Terminalbefehle für die ersten Tage vor dem Unix-Vortrag.
```

File: terminal-cheat-sheet/content/cheat-sheet.tex [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+\begin{frame}{Cheat-Sheet}
+  \begin{tblr}{
+      colspec = {rl},
+      column{1} = {font=\ttfamily, fg=vertexDarkRed}
+    }
+    ls & \enquote{list}: Zeigt den Inhalt eines Verzeichnisses \\
+    cd .. & \enquote{change directory} wechselt in den Oberordner \\
+    cd - & \enquote{change directory} in das vorherige Verzeichnis \\
+    cd <dirname> & \enquote{change directory} wechselt in den angegebenen \\
```
Comment (jpherdi) [inline terminal-cheat-sheet/content/cheat-sheet.tex:11]:
```suggestion
    cd <dirname> & \enquote{change directory} wechselt in das Verzeichnis <dirname> \\
```

Comment (jpherdi) [reply terminal-cheat-sheet/content/cheat-sheet.tex:11]: Würde ich als erstes schreiben und dann `cd ..` und `cd -` als Spezialfälle zeigen

File: terminal-cheat-sheet/content/cheat-sheet.tex [hunk 5]
```diff
@@ -0,0 +1,16 @@
+\tabulinesep=4pt
+
+\begin{frame}{Cheat-Sheet}
+  \begin{tblr}{
+      colspec = {rl},
+      column{1} = {font=\ttfamily, fg=vertexDarkRed}
+    }
+    ls & \enquote{list}: Zeigt den Inhalt eines Verzeichnisses \\
+    cd .. & \enquote{change directory} wechselt in den Oberordner \\
```
Comment (jpherdi) [inline terminal-cheat-sheet/content/cheat-sheet.tex:9]:
```suggestion
    cd .. & wechselt in das Oberverzeichnis \\
```

File: terminal-cheat-sheet/content/cheat-sheet.tex [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    cd - & \enquote{change directory} in das vorherige Verzeichnis \\
```
Comment (jpherdi) [inline terminal-cheat-sheet/content/cheat-sheet.tex:10]:
```suggestion
    cd - & wechselt in das vorherige Verzeichnis \\
```

**Review by jpherdi — CHANGES_REQUESTED** [review]

**Pull request merged**